src/head/Column.js

```diff
@@ -11,25 +9,30 @@ export function Column({column, className, Element}) {
         className
     };
 
-    if(column.onSort) {
-        props.onClick = () => {
-            column.onSort(column);
-        };
-    }
-
     return (
         <Element {...props}>
-
             {column.title}
-
-            <Sorter
-                sorted={column.sorted}
-                sorterComponent={column.sorterComponent}
-            />
+            {renderSorter(column)}
         </Element>
     );
 }
 
+function renderSorter(column) {
+    return column.isSortable ? (
+        <Sorter
+            onClick={() => onColumnClick(column)}
+            sorted={column.sorted}
+            sorterComponent={column.sorterComponent}
+        />
+    ) : null;
+}
+
+function onColumnClick(column) {
+    if (column.onSort) {
+        column.onSort(column);
+    }
+}
```

Comment:
You have changed default behavior. Now onClick is defined on the sorter, not on the entire header cell. Is this intended? You have to click exactly on the sorter to sort column.